### ISSUE TITLE
Allow usage of global `session.unsubscribe` to unsubscribe from individual contexts

### ISSUE THREAD

**Issue by lutien** [w3c/webdriver-bidi#326]
At the moment, a client can send global `session.unsubscribe` command only if it sent before global `session.subscribe`, otherwise it gets an invalid argument error. Which would mean if a client subscribes to individual contexts, it would need to keep track of the contexts in order to unsubscribe from them later.
To make it easier for the clients, the proposal would be to allow usage of global `session.unsubscribe` to unsubscribe from individual contexts.

**Comment by juliandescottes** [comment]
Can we also discuss about the non-global scenario? 

If we unsubscribe from a specific event, on a specific context, we also throw an invalid argument error if we were not previously subscribed to the event. This would make cleanup code harder to write because you need to keep track of your event subscriptions.

**Comment by whimboo** [comment]
Yes, that's a good addition. I've updated the summary and will add an agenda item for tomorrow's meeting.
